getting-started/display.rst

```diff
@@ -172,7 +172,7 @@ Then use the display's ``mtv`` method to view the ``calexp`` in DS9:
 
 Notice that the DS9 display is filled with colorful regions.
 These are mask regions.
-Each color reflects a different mask bit that correspond to detections and different types of detector artifacts.
+Each color reflects a different mask bit that corresponds to detections and different types of detector artifacts.
```

Comment:
"different mask bit that corresponds to detections" could be misread as "each detection has its own mask bit" or something similarly off-track.

Maybe:

Each color corresponds to one of a common set of boolean flags defined for each pixel; each flag has a distinct meaning, covering a range of concepts from detector artifacts to the contribution of a pixel to a detection.  In this project we call the array of these flags the "mask" or "mask plane" associated with an image.  (NB: This is a distinct  concept from a "mask" for an array in ``numpy``.)  You'll learn later how to interpret these flags (and their colors); for now you'll likely want to adjust the image display itself.

Reply:
Thanks for the suggested rewording! I've implemented this in [c220b3a](https://github.com/lsst/pipelines_lsst_io/pull/292/commits/c220b3addec6cc7bbe57d3b2924a6109d0c1bed2)